src/main/java/gov/cms/madie/services/PackagingService.java

```diff
@@ -82,12 +82,18 @@ public byte[] createQRDA(QrdaRequestDTO qrdaRequestDTO, String accessToken) {
         qrdaExport.getSummaryReport().getBytes());
     for (QrdaReportDTO qrda : qrdaExport.getIndividualReports()) {
       if (qrda.getQrda() == null) {
-        log.warn("QRDA export for Measure [{}], testcase filename [{}] is missing QRDA", measure.getId(), qrda.getFilename());
+        log.warn(
```

Comment:
do we collect this logging level? I know we do info and error

Reply:
Idk what log levels we are collecting, if we already not collecting warn, then I think we should, Because in this use case warn makes sense.

Reply:
I can check with @sb-bdonohue